src/Main.java

```diff
@@ -1,7 +1,56 @@
+import java.util.Arrays;
+
 //TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
 // click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
     public static void main(String[] args) {
 
+        //Task#1
+        int [] array = {5000, 3500, 4200, 3700, 2900};
+        int count = 0;
+        for (int i = 0; i < array.length; i++) {
+            count += array[i];
+        }
+        System.out.println("Сумма трат за меся составила " + count + " рублей");
+
+        System.out.println();
+        //Task#2
+
+        int minimalValue = 99999;
+        int maximumValue = -1;
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] > maximumValue) {
+                maximumValue = array[i];
+            }
+            if (array[i] < minimalValue) {
+                minimalValue = array[i];
+            }
+        }
+        System.out.println("Максимальная сумма трат за неделю составила " + maximumValue + " рублей");
+        System.out.println("Минимальная сумма трат за неделю составила " + minimalValue + " рублей");
+
+        System.out.println();
+        //Task#3
+
+        float count1 = 0;
```

Comment:
нет смысла заводить новую переменную с плохим названием (когда на конце цифры), когда такая переменная уже была объявлена выше
```suggestion
        count = 0;
```